Submitted By: Ryan Oliver <ryan.oliver@pha.com.au>
Date: 2005-08-06
Initial Package Version: 1.1.10
Upstream Status: From Upstream
Origin: Gentoo (via speex CVS)
Description: fix build when using sse

--- speex-1.1.10/libspeex/vq.c	2005-06-04 13:57:12.000000000 +1000
+++ speex-cvs/libspeex/vq.c	2005-07-24 01:42:50.000000000 +1000
@@ -40,6 +40,7 @@
 
 #ifdef _USE_SSE
 #include <xmmintrin.h>
+#include "vq_sse.h"
 #elif defined(SHORTCUTS) && (defined(ARM4_ASM) || defined(ARM5E_ASM))
 #include "vq_arm4.h"
 #elif defined(BFIN_ASM)
